buildbot/tox_setup.sh

```diff
@@ -1,74 +1,102 @@
 #!/usr/bin/env bash
-# pyenv root
-export PYENV_ROOT="$HOME/.pyenv"
-TEST_ROOT=$PWD/..
 
-# Install pyenv if it's missing
-if [[ ! -d $PYENV_ROOT ]]; then
-    git clone git://github.com/yyuu/pyenv.git ${PYENV_ROOT}
-    cd ${PYENV_ROOT}
-    # Get the latest tagged version
-    git checkout `git tag | tail -1`
+if [[ ! -d $PYENV_ROOT ]]
+then
+    export PYENV_ROOT="$HOME/.pyenv"
 fi
 
-# Upgrade it, if it's too old
-if [[ -z $(pyenv install --list | grep 3.4.3) ]]; then
-    cd ${PYENV_ROOT}
-    git pull origin master
-    git pull -u origin master
-    # Get the latest tagged version
-    git checkout `git tag | tail -1`
+declare -r PROJDIR="$PWD/.."
+if [[ ! -s $PROJDIR/riak/__init__.py ]]
+then
+    echo "[ERROR] script must be run from the root of a clone of github.com/basho/riak-python-client" 1>&2
+    exit 1
 fi
 
-if [[ ! -d ${PYENV_ROOT}/plugins/pyenv-virtualenv ]]; then
-    git clone https://github.com/yyuu/pyenv-virtualenv.git ${PYENV_ROOT}/plugins/pyenv-virtualenv
-    cd ${PYENV_ROOT}/plugins/pyenv-virtualenv
-    git checkout `git tag | tail -1`
+if [[ ! -d $PROJDIR/riak_pb/src ]]
+then
+    (cd $PROJDIR && git submodule update --init)
```

Comment:
To be honest I added this more for someone who is running `tox_setup.sh` standalone. I'll take this out and if / when someone runs into the missing submodule when running `pb_build` they'll figure it out fast enough.